src/components/ClickableStyle/ClickableStyle.module.css

```diff
@@ -454,21 +454,25 @@
 
 /**
  * Link button styles
+ * 1) Override default Clickable styles
+ * 2) Smaller gap between text and icon in link button style
  */
 .clickable-style--link {
   @mixin textLink;
 
-  border-radius: 0;
+  /* 1 */
   border: 0;
+  border-radius: 0;
   background-color: transparent;
+
+  gap: 0.25em; /* 2 */
```

Comment:
Would the comments be easier to make sense of if they were inline / next to the thing they're commenting?

I haven't looked at EDS code in a little while, but this style of referencing another point in this file feels harder for me to put understand.

Also may be less likely to update or remove comments if they're next to the commentee.

Reply:
I think they would be able to understand, yes. I've seen a few times where outdated comments hung around after the code below it had changed, and I do think that would be less likely if the comment were next to the code. This format came from Big Medium. cc @jinlee93 @anniehu4

Reply:
very pro ditching this commenting style and going back to having things right next to what they're referencing :')

Reply:
https://app.shortcut.com/czi-edu/story/211537/move-css-comments-next-to-what-they-re-referencing

Reply:
Next time we work with a contractor they gotta go through some training (roles and responsibilities, quality standards, expectations, etc).